client/web/src/enterprise/batches/preview/list/PreviewSelectRow.tsx

```diff
@@ -0,0 +1,112 @@
+import { noop } from 'lodash'
+import InfoCircleOutlineIcon from 'mdi-react/InfoCircleOutlineIcon'
+import React, { useMemo, useContext } from 'react'
+
+import { pluralize } from '@sourcegraph/shared/src/util/strings'
+import { useObservable } from '@sourcegraph/shared/src/util/useObservable'
+
+import { BatchSpecApplyPreviewVariables } from '../../../../graphql-operations'
+import { Action, DropdownButton } from '../../DropdownButton'
+import { MultiSelectContext } from '../../MultiSelectContext'
+
+import { queryPublishableChangesetSpecIDs as _queryPublishableChangesetSpecIDs } from './backend'
+
+const ACTIONS: Action[] = [
+    {
+        type: 'unpublish',
+        buttonLabel: 'Unpublish',
+        dropdownTitle: 'Unpublish',
+        dropdownDescription:
+            'Do not publish selected changesets on the codehost. Note, we cannot unpublish a published changeset.',
```

Comment:
Slight wording suggestion (but I defer to @rrhyne if he has thoughts):

```suggestion
            'Do not publish selected changesets on the codehost. Note that a changeset that has already been published cannot be unpublished.',
```

Reply:
It's funny, I actually originally had almost literally this, but it does make the description extend onto a third line:

<img src="https://user-images.githubusercontent.com/8942601/129656291-63927a3b-1af8-4261-b6f2-043edb9f484f.png" width="400">

It's probably okay since the other two action descriptions are so short?

Reply:
Slightly shorter: 

Note: a changeset that has been published cannot be unpublished.

Reply:
Much better:
![image](https://user-images.githubusercontent.com/8942601/129778115-ed3a0a5b-697e-4b62-9232-e20a23d3acd4.png)